### PR TITLE
CPU temperature measuring

### LINKED ISSUE #156: Check the temperature of CPU using Python
**Is your feature request related to a problem? Please describe.**
Title

**Describe the solution you'd like**
Try using Psutil for a solution

### PR DESCRIPTION
**Pull request by imakash3011**
Fixes #156

### Checklist:
- [x] My branch is up-to-date with upstream/master branch.

### Description:

Added python code for measuring CPU temperature.

### CODE REVIEW

**Comment by Tejas1510** [comment]
@tripathyprateek please give a look
As this was created by you, so you will better understand the code

**Comment by Tejas1510** [comment]
@tripathyprateek what is the status
@atarax665 else you have a look

File: Python/CPU_Temperature/temperature.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import psutil
+
+# Not for windows user 
+data = psutil.sensors_temperatures()
+print("Current Temperature of CPU (celcius): ",data['coretemp'][0][1])
```
Comment (atarax665) [inline Python/CPU_Temperature/temperature.py:5]: Add a blank line at the end of file.

**Review by atarax665 — COMMENTED** [review]
Add a proper Readme file.

File: Python/CPU_Temperature/temperature.py (hunk 1 above)
Comment (imakash3011) [reply Python/CPU_Temperature/temperature.py:5]: Okay @atarax665

**Pull request closed without merging**